shell.nix

```diff
@@ -0,0 +1,34 @@
+let
+  sources = import ./nix/sources.nix;
+  pkgs = import sources.nixpkgs { };
+
+  stack-nix = pkgs.symlinkJoin {
+    name = "stack";
+    paths = [ pkgs.stack ];
+    buildInputs = [ pkgs.makeWrapper ];
+    postBuild = ''
+      wrapProgram $out/bin/stack \
+        --add-flags "\
+          --nix \
+          --no-nix-pure \
+          --nix-shell-file=nix/stack-integration.nix \
+        "
+    '';
+  };
+in
+pkgs.mkShell {
+  name = "tagger-dev";
+  NIX_PATH = "nixpkgs=" + pkgs.path;
+  buildInputs = with pkgs; [
+    elmPackages.elm
+    elmPackages.elm-language-server
+    haskellPackages.haskell-language-server
+    haskellPackages.ormolu
```

Comment:
this project is not formatted with `ormolu`, do we actually need it?

Reply:
:shrug:

Counter-question: should this project be formatted with `ormolu`/anything else?

Reply:
I vote for `ormolu` or (even better in my "heretic" opinion) `fourmolu`.

Reply:
I am open to try both, but if we add them, they need to be documented and the code needs to be formatted with the chosen tool

Reply:
I'll do it in a different pull request! :)

Reply:
could you remove it from here, then, and readd it in the new PR?